### PR TITLE
Added LMGTFY command.

### PR DESCRIPTION
**Pull request by harshhh-dev**
Sends  a link for the user which sends them the answer.

### CODE REVIEW

**Comment by lenscas** [comment]
I rather have the google command to do the search instead of LMGTFY. #23 :)

**Comment by harshhh-dev** [comment]
Ok. I'll replace it then.

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,5 @@ yarn-error.log
 database.json
 docs
 build
-config.json
\ No newline at end of file
+config.json
+package-lock.json
```
Comment (lenscas) [inline .gitignore:8]: what is this file and why is it suddenly inside the .gitignore file?

File: src/commands/oftenAsked/google.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { create_command } from '../../command';
+
+export const command = create_command(async ({ message , args}) => {
+    const search = args.join('+')
+
+    message.channel.send(`Find your answer at <https://www.google.com/search?q=search%3F%3F%3F%3F&oq=ho&aqs=chrome.0.69i59l3j35i39j69i57j69i60l3.7034j0j1&sourceid=chrome&ie=UTF-8>!`)
```
Comment (lenscas) [inline src/commands/oftenAsked/google.ts:6]: it doesn't use the search parameter? Also I'm pretty sure that those other get parameters can go.

**Review by lenscas — CHANGES_REQUESTED** [review]

**Review by lenscas — APPROVED** [review]

**Pull request merged**